elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/integration/AggregationDataStoreIntegrationTest.java

```diff
@@ -749,6 +748,24 @@ public void metaDataTest() {
                 .body("data.attributes.expression",  equalTo("player.name"))
                 .body("data.relationships.table.data.id", equalTo("playerStats"));
 
+        given()
```

Comment:
This looks like a separate test.  Let's not complicate existing tests by joining them together.  Also, there are already metadata tests - let's see how this new test relates to those.